libs/visor_dns/dns.h

```diff
@@ -28,6 +28,7 @@ enum RCode {
 };
 
 static std::unordered_map<uint16_t, std::string> QTypeNames({
+    {0, "Reserved"},
```

Comment:
Would it make sense to label this "Reserved (0)" to help differentiate from the other reserved qtypes?

Reply:
yes, fixed